languages/common-lisp/reference/exercise-concepts/word-count.md

```diff
@@ -0,0 +1,39 @@
+# General Concepts of Word Count
+- Variables: Local variables with `let` or `let*` are commonplace
+- String: Strings need to be split into a list of words. Cleaning is also done
+  with `string-trim` and `string-downcase`
+- Functions: Helper functions are commonly defined
+- Sameness: Common lisp has a _lot_ of equality operators. Most useful here are:
+  `equal` and `string=`
+- Comments: We can hope for well commented code (this includes the use of
+  docstrings!)
+- Nested Functions: Often appear as string cleaners or serve other
+  functions. Defined with `flet` or `labels`
+- Higher Order Functions: The use of various functions like `remove-if` and
+  `map` is common
+- Anonymous Functions: Some students may pass a `lambda` to functions like `map`
+- Lists: This exercise requires a result given as an `alist`
+- Boolean Logic: Compositions of `and` and `or`. Helpfully short-circuiting
+
+# String -> Words
+## Approach: Library Import
+- Regular Expressions: Needed if the student is using the `cl-ppcre`
```

Comment:
I still hold the stance of *not* having students use 3rd party libraries.

Reply:
I also somewhat agree with that, but looking through the solutions to this exercise, many people did adopt this approach, so I'm a bit stuck on the fence personally.

I agree that we should be teaching the language, but there are also some places (perhaps we could isolate them under a super-concept of "Popular Libraries" that show off some common add-ons (e.g. UIOP, Alexandria maybe, etc)?